_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -1198,7 +1225,7 @@ sections above.
 1.18,<<iam_roles_for_instances>>,Use Gruntwork modules to ensure EC2 instances use roles for access
 1.19,TODO(Rob),TODO
 1.20,TODO(Ana),TODO
-1.21,TODO(Ina),TODO
+1.21,<<iam_access_analyzer>>,Use Gruntwork modules to enable IAM Access Analyzer across regions
```

Comment:
This does not render correctly - perhaps it needs to be `<<iam_access_analyzer_deployment>>`?

Reply:
ahh of course - I renamed it!